### ISSUE TITLE
Incomplete GTA5 to Cityscapes segmentation code and low mIoU

### ISSUE THREAD

**Issue by akshaykulkarni07** [NVlabs/CSG#2]
Hi, 
Congratulations on your great work and thanks for releasing the code.

For the GTA5 to Cityscapes training code, I could not find the `gta5_train.txt` file. While I created it on my own, the code also does not map the GTA5 labels from 1-33 range to the Cityscapes label IDs (0-18) which leads to errors. I managed to fix that by performing the mapping. However, after training for around 4-5 epochs, the Cityscapes validation mIoU is very low (~0.5 to 1% only) and not increasing. I find this very odd since plain training on GTA5 usually quickly gives at least 15-25% mIoU. There is no model specified as initialization and I believe the PyTorch ImageNet-pretrained one is being used.

Kindly advise on whether some other initialization is required or some code additions are required or some preprocessing is required for the datasets. It would be very helpful if you could release the corrected code.

**Comment by akshaykulkarni07** [comment]
The `cv2.imread` for the label did not read the values properly. After changing it to `Image.open` (PIL) and using remapping, it is giving higher mIoUs (~30%) at the initial epochs.
